atsign-foundation/at_wavi_app
# fix: fixed issues with upgrading to newer sdk

**- What I did**
I upgraded android SDK version to 34

**- How I did it**
I changed it in build.gradle file and fixed the issues that followed up

**- How to verify it**
Crossverifying it with checking out build.gradle file would work and cloning it into local machine and running the application will check that

**- Description for the changelog**

Upgraded android SDK to version 34 that was required by google play store and fixed the issues that followed
